src/openrct2/world/TileInspector.cpp

```diff
@@ -249,9 +249,32 @@ sint32 tile_inspector_rotate_element_at(sint32 x, sint32 y, sint32 elementIndex,
             tileElement->properties.path.edges |= ((pathEdges << 1) | (pathEdges >> 3)) & 0x0F;
             tileElement->properties.path.edges |= ((pathCorners << 1) | (pathCorners >> 3)) & 0xF0;
             break;
+        case TILE_ELEMENT_TYPE_ENTRANCE:
+        {
+            // Update element rotation
```

Comment:
This does mean that lowering/raising an entrance building will automatically do 'Make usable' as well, in practical terms.

Reply:
Now with 4ca07c19c0d876e9fe5dd244b9d989ff8d722993 it only updates the rotation when the element's location matches the known location.